src/main/angular/app/services/real-services/http.service.ts

```diff
@@ -0,0 +1,22 @@
+import {Injectable} from '@angular/core';
+import {HttpClient} from '@angular/common/http';
+import {ProjectGraphData} from '../../../model/project-graph-data';
+import {Project} from '../../../model/project';
+import {DataService} from '../data.service';
+
+@Injectable()
+export class HttpService implements DataService {
```

Comment:
Consider giving this a different name (e.g. DataServiceImpl), because as this project grows you'll likely want several different services that can make http requests.